packages/indexer-client/src/types/clientTypes.ts

```diff
@@ -253,9 +253,9 @@ export type GetIndexerMultiProductSnapshotsResponse = Record<
 >;
 
 export interface IndexerSnapshotsIntervalParams {
-  // Currently accepts all integers, in seconds
+  /**  Currently accepts all integers, in second */
   granularity: number;
-  // Seconds
+  /** In seconds. If maxTimeInclusive is not set, the query may return the earlier timestamps near midnight UTC.*/
```

Comment:
`the query may return the earlier timestamps near midnight UTC` what does this mean?

Reply:
Would this make more sense
```
  /**
   * Optional upper bound for snapshot timestamps (in seconds).
   * Without this, snapshots may default to align with UTC midnight,
   * which can make "Last 24h" metrics inaccurate.
   */
```